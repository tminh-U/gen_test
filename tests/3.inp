12
9 5 19 15 13 3 14 15 8 1 16 4 
12
6 12
11 11
1 10
8 9
7 10
6 9
1 1
7 12
11 11
4 7
8 10
1 3
