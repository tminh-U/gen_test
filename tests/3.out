25
2
41
3
6
10
2
14
2
7
4
6
